SPECS/tdnf/tdnf-default-mariner-release.patch

```diff
@@ -14,23 +13,17 @@ mariner-release.noarch                      1.0-1.cm1                @System
 
 Workaround: add the following to /etc/tdnf/tdnf.conf
 distroverpkg=mariner-release
----
- client/defines.h | 2 +-
- 1 file changed, 1 insertion(+), 1 deletion(-)
 
 diff --git a/client/defines.h b/client/defines.h
-index bee38d6..cce8862 100644
+index 7cb16b8..5131040 100644
 --- a/client/defines.h
 +++ b/client/defines.h
-@@ -143,7 +143,7 @@ typedef enum
- //Repo defaults
+@@ -121,7 +121,7 @@ typedef enum
+ // repo defaults
  #define TDNF_DEFAULT_REPO_LOCATION        "/etc/yum.repos.d"
  #define TDNF_DEFAULT_CACHE_LOCATION       "/var/cache/tdnf"
--#define TDNF_DEFAULT_DISTROVERPKG         "photon-release"
+-#define TDNF_DEFAULT_DISTROVERPKG         "system-release"
```

Comment:
Do we need to keep maintaining this patch? Our `mariner-release` package has a provides for `system-release`, so I suspect we can go without.

Reply:
We could also provide an entry in tdnf.conf. I would need to verify this. I would happily drop it (maintaining this patch is trivial), but we can also do that in the next iteration.

Reply:
Let's keep the patch.  Presumably you've already ran a build and tested, etc.  Minimize the amount of re-work here, and keep the change risk at a minimum.